lib/geojson.js

```diff
@@ -11,7 +11,8 @@ function GeoJSON(filepath) {
     this.datasource = new mapnik.Datasource({
       type: 'geojson',
       file: filepath,
-      cache_features: false
+      cache_features: false,
+      num_features_to_query: 1000000
```

Comment:
@mapsam Thanks for the question. I hardcoded this number based on the 1.3GB test file I used, which had 710,415 features. We can make adjustments down the road, or make it an argument/option if warranted.